app.ts

```diff
@@ -0,0 +1,462 @@
+const headings: any = document.querySelector("#Headings");
+const content_cols: any = document.querySelector("#Content");
+const pageStats: any = document.getElementById("pageStats");
+const nextButton: any = document.querySelector("#next");
+const prevButton: any = document.querySelector("#prev");
+const clear = "";
+let paramOne: any = "0";
+let paramTwo: any = "9";
+let contentNeeded: any = [];
+
```

Comment:
There are too many global variables here which is generally frowned upon. There are few variables that are acceptable as global variables.